tests/build.gradle

```diff
@@ -13,6 +13,8 @@ dependencies {
 }
 
 android {
+  namespace "com.google.android.material"
```

Comment:
I've used the package name defined here:
https://github.com/material-components/material-components-android/blob/c30c2d8056bc9b12bd118f0fe55744198ac3738c/tests/javatests/com/google/android/material/AndroidManifest.xml#L19